fe1-web/src/features/evoting/objects/__tests__/Election.test.ts

```diff
@@ -53,14 +55,15 @@ const initializeData = () => {
 
   electionState = {
     id: 'electionId',
-    eventType: EventTypeElection,
+    eventType: LaoEventType.ELECTION,
```

Comment:
there was a reason for not using `LaoEventType`, but it's clearly not obvious from the current code-base.

One of the goal of the current separation into features is that each feature should be as independent as possible.

Ideally, we would like `meeting`, `rollCall` and `election` to each have their own reducer, + an `events` reducer which is type-agnostic and just allows other features to associate their own keyword (e.g. `EventTypeElection`) to a given way of displaying the event. 
This would allow us to reach the state where features do not import each other *at all*. 
Remind me to talk about this in one of our next meetings.